packages/astro/src/runtime/client/dev-overlay/entrypoint.ts

```diff
@@ -1,5 +1,5 @@
 import type { DevOverlayPlugin as DevOverlayPluginDefinition } from '../../../@types/astro.js';
-import { type AstroDevOverlay, type DevOverlayPlugin } from './overlay.js';
```

Comment:
This line was the motivation for this, the empty import to `overlay.js` was causing it to load early into the page, causing performance issue even on just my small website.